constants.yml

```diff
@@ -14,6 +14,7 @@ jurisdiction:
     'england' : "England"
     'noneu' : "Foreign (Non E.U.)"
 company_summary:
+    'registered-overseas-entity' : "Registered overseas entity"
```

Comment:
I don't remember how these enumerations are supposed to work. In the absence of any other ordering constraints, I would be inclined to place new values at the end of each section. Probably does not matter?

Reply:
I'm not sure either but I do agree, I think these new values should be placed at the end of each section. Looking at previous PR's this is what has been done.